winui/DataGrid/Autosize-Columns.md

```diff
@@ -123,6 +123,63 @@ this.sfDataGrid.Columns["OrderID"].ColumnWidthMode = ColumnWidthMode.AutoLastCol
 
 <img src="Autosize-Columns-images/winui-datagrid-column-filling.png" alt="Specific Column in WinUI DataGrid Fills Remaining Width" width="100%" Height="Auto"/>
 
+## Improving Performance of Auto-Sizing Calculations
+
+By default, column auto-sizing is calculated for all rows in the DataGrid. The `AutoFitRange` property allows you to specify whether auto-sizing is based on visible rows or all rows in the DataGrid. The default value for the `AutoFitRange` property is AllRows.
+
+The column width adjusts based on the visible text and maintains the maximum width of the column.
+
+Below are the options available for AutoFitRange:
+
+<table>
+<tr>
+<th>
+Type
+</th>
+<th>
+Description
+</th>
+</tr>
+<tr>
+<td>
+<code>VisibleRows</code>
+</td>
+<td>
+Specifies that column auto-sizing considers only the visible rows in the grid.
+</td>
+</tr>
+<tr>
+<td>
+<code>AllRows</code>
+</td>
+<td>
+Specifies that column auto-sizing considers all rows in the grid.
+</td>
+</tr>
+</table>
+
+{% tabs %}
+{% highlight xaml %}
+<dataGrid:SfDataGrid x:Name="sfDataGrid"
+                        AutoGenerateColumns="False"
+                        ColumnWidthMode="AutoLastColumnFill"
+                        AutoFitRange="VisibleRows"
+                        ItemsSource="{Binding OrdersDetails}">            
+            <dataGrid:SfDataGrid.Columns>
+                <dataGrid:GridTextColumn HeaderText="Order ID" MappingName="OrderID" ColumnWidthMode="AutoLastColumnFill" />
+                <dataGrid:GridTextColumn HeaderText="Customer ID" MappingName="Customer ID" />               
```

Comment:
Why there is a space in mapping name?

Reply:
@sethuramkumar - Okay , I have modified the xaml code.